components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -770,36 +771,42 @@ class PageWallet extends React.Component<Props, State> {
 
     return (
       <>
-        <WalletWrapper
-          balance={total.toFixed(1)}
-          converted={utils.formatConverted(this.getConversion())}
-          actions={this.getActions()}
-          onSettingsClick={this.onModalBackupOpen}
-          showCopy={showCopy}
-          showSecActions={true}
-          grants={this.generatePromotions()}
-          alert={this.walletAlerts()}
-          walletState={this.getWalletStatus()}
-          onVerifyClick={onVerifyClick}
-          onDisconnectClick={this.onDisconnectClick}
-          goToUphold={this.goToUphold}
-          userName={this.getUserName()}
-          onlyAnonWallet={onlyAnonWallet}
-        >
-          {
-            enabledMain
-            ? emptyWallet && pendingTotal === 0
-              ? <WalletEmpty onlyAnonWallet={onlyAnonWallet} />
-              : <WalletSummary
-                reservedAmount={pendingTotal}
+        {
+          externalWallet
```

Comment:
Only change here is to not show the wallet wrapper until we have a valid external wallet.